Source/ThirdParty/ANGLE/extensions/EGL_ANGLE_sync_mtl_shared_event.txt

```diff
@@ -0,0 +1,56 @@
+Name
+
+    ANGLE_metal_shared_event_sync
+
+Name Strings
+
+    EGL_ANGLE_metal_shared_event_sync
+
+Contributors
+
+    Dan Glastonbury, Apple
+
+Contacts
+
+    Dan Glastonbury, Apple (djg 'at' apple.com)
+
+Status
+
+    Draft
+
+Version
+
+    Version 1, 2022-05-17
+
+Number
+
+    EGL Extensions XXX
+
+Extension Type
+
+    EGL display extension
+
+Dependencies
+
+Overview
```

Comment:
Could you add a couple of sentences describing the intent and how to use it?

Reply:
I planned to do this when upstreaming to ANGLE.

Reply:
@kenrussell https://bugs.chromium.org/p/angleproject/issues/detail?id=7561